src/index.d.ts

```diff
@@ -223,10 +223,10 @@ declare namespace preact {
 		parent: Element | Document | ShadowRoot | DocumentFragment
 	): void;
 	function cloneElement(
```

Comment:
We could allow a generic here as we use [in createElement](https://github.com/preactjs/preact/pull/2388/files#diff-b739f95ace6bbf2f0fd5e8cf1dab97d8R185)